quickstart.rst

```diff
@@ -51,7 +67,37 @@ Build flux-core. In order to build python bindings, ensure you have python-3.6 a
 
 .. note::
    Flux still supports python-2.7, but we recommend that you use python-3.6 or higher
-   as the Python community will stop maintaining this version in 2020.
+   as the Python community has stopped maintaining this version as of 2020. The
+   ``PYTHON_VERSION`` environment variable adds a suffix to the python interpreter
+   executable. Configure would look for python3 in the example above.
+
+Ensure all is right with the world by running the built-in ``make check`` target:
+
+.. code-block:: console
+
+  $ make check
+  Making check in src
+  ...
+
+Clone current flux-sched master:
+
+.. code-block:: console
+
+  $ git clone https://github.com/flux-framework/flux-sched.git
+  Initialized empty Git repository in /g/g0/grondo/flux-sched/.git/
+  $ cd flux-sched
+
+Build flux-sched:
+
+.. code-block:: console
+
+  $ ./autogen.sh && PYTHON_VERSION=3 ./configure
```

Comment:
One comment here (and apologies if I misread because I was going quickly) -- this will configure flux-sched against a system installed flux-core, i..e `pkg-config` will find the `flux-core.pc` from default `PKG_CONFIG_PATH`.

My typical workflow is to install flux-core first to prefix `/usr/local` (maybe suggest `$HOME/local` if users don't have write access to system directories), then `PKG_CONFIG_PATH=$HOME/local/lib/pkgconfig PYTHON_VERSION=3 ./configure` ... 

Perhaps we need to make this easier by adding a `--with-flux-core-prefix=PREFIX` to `flux-sched`'s configure?